File: Fundamentals-of-Programing141-main/homework/02_variables_expressions/02_intermediate/04_back_pay/back_pay.py
"""
Intermediate Problem:  Create a program that outputs the amount of retroactive pay due
                       the employee, the new annual salary, and the new monthly salary.

File Name: back_pay.py
Name:      ?
Course:    CPTR 141
"""

RAISE = 0.076

# Your code goes here
annual_sal = float(input("Enter current annual salary: $"))
num_month_back_pay = float(input("Enter number of months for back pay: "))

print("\nI'll return new annual salary, monthly salary, and retroactive pay in dollars.")
n_annual_sal = annual_sal * (1+RAISE)
print("New annual salary:" + "{:12.2f}".format(n_annual_sal))
n_month_sal = n_annual_sal/12
print("New monthly salary:" + "{:11.2f}".format(n_month_sal))
month_extra = n_month_sal - (annual_sal/12)
retra_pay = month_extra * num_month_back_pay
print("Retroactive pay:" + "{:14.2f}".format(retra_pay))

File: Fundamentals-of-Programing142-main/homework/objective_03/4pt_back_pay/back_pay.cpp
/*************************************************************************
 *
 * Objective 4: Determining an employee's retroactive pay.
 *
 * File Name: back_pay.cpp
 * Name:      ?
 * Course:    CPTR 142
 * Date:
 *
 */

#include <iomanip>
#include <iostream>
#include <ostream>

int main() {
  // Define your variables
  double annual_salary;
  double num_months;
  const double percent_increase = 1.076;
  double new_salary_year;
  double new_salary_month;
  double retroactive_due;
  // Collect user input
  std::cout << "Enter current annual salary: ";
  std::cin >> annual_salary;
  std::cout << "Enter number of months for back pay: ";
  std::cin >> num_months;
  // Perform calculations
  new_salary_year = annual_salary * percent_increase;
  new_salary_month = new_salary_year / 12;
  retroactive_due =
      (new_salary_month * num_months) - ((annual_salary / 12) * num_months);

  // Print out the results
  std::cout << std::fixed << std::setprecision(2);
  std::cout
      << std::endl
      << "I'll return new annual salary, monthly salary, and retroactive pay."
      << std::endl;
  std::cout << "New annual salary: $" << new_salary_year << std::endl;
  std::cout << "New monthly salary: $" << new_salary_month << std::endl;
  std::cout << "Retroactive salary due: $" << retroactive_due << std::endl;
  return 0;
}
